docs/src/filters.md

```diff
@@ -89,6 +89,74 @@ ZstdFilter
 
 Additional filters can be dynamically loaded by the HDF5 library. See [External Links](@ref) below for more information.
 
+### Using an UnknownFilter (aka ExternalFilter)
```

Comment:
Let's switch this around and have the docs mainly refer to `ExternalFilter` rather than `UnkownFilter`